miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/ListKitDescriptorsController.java

```diff
@@ -74,7 +74,7 @@ public ModelAndView listKitDescriptors(ModelMap model) throws IOException {
       return new ModelAndView("/pages/listKitDescriptors.jsp", model);
     } catch (IOException ex) {
       if (log.isDebugEnabled()) {
-        log.debug("Failed to list Kit Descriptors", ex);
+        log.debug("Failed to list KitComponent Descriptors", ex);
```

Comment:
This should still be `Kit Descriptors` since it's the `ListKitDescriptorsController`, no?